src/main/java/lotto/LottoPrize.java

```diff
@@ -0,0 +1,34 @@
+package lotto;
+
+import java.util.Arrays;
+
+public enum LottoPrize {
+
+    NONE(0,0),
+    MATCH_3(3, 5000),
```

Comment:
자바에서는 숫자의 가독성읖 높히기 위해 언더바(`_`)를 허용합니다. 
```suggestion
    MATCH_3(3, 5_000),
```

Reply:
이런 기능이 있을줄이야!! 감사합니다.